README.md

```diff
@@ -1,7 +1,9 @@
 ## Test262 is the official ECMAScript (ECMA-262) conformance suite.
 
+Test262 tests conformance to the continually maintained draft future ECMAScript standard found at http://tc39.github.io/ecma262/ , together with any Stage 3 or later TC39 proposals. It is maintained by Gorkem Yakin (@goyakin) with significant contributions from many in the ECMAScript community.
+
```

Comment:
Sorry if I got this wrong. I was trying to document the state of the world, especially since the old test262 site is gone. As for test262 being a reflection of the draft spec, that was agreed at the September TC39 meeting https://github.com/rwaldron/tc39-notes/blob/dd8bb263aef908509c91f58e4912482a75c1c74d/es7/2015-09/sept-24.md#8-test262-update . For Gorkem being the maintainer--are you another maintainer, or are there other, additional maintainers? I apologize for any omission. Brian Terlson used to be a maintainer of test262, but he delegated this to Gorkem recently. It's useful to have maintainers listed here so that people can bug a maintainer if they aren't getting a code review.